src/components/text/_headers.scss

```diff
@@ -1,50 +1,39 @@
-$headerColor: $black;
-$headerColorLight: $white;
-$headerPrimaryFontFamily: $fontFamilyEmphasized;
-$headerSecondaryFontFamily: $fontFamilySecondary;
-$headerPrimaryFontSize: 64px;
-$headerPrimarySmallFontSize: 48px;
-$headerSecondaryFontSize: 16px;
-$headerSecondarySmallFontSize: 12px;
-
 $include-html: false !default;
 
 @if ($include-html) {
 
   .mint-header-primary {
     @include component;
+    @include typeVariant(header, 2);
     display: block;
-    color: $headerColor;
-    font-family: $headerPrimaryFontFamily;
-    font-size: $headerPrimaryFontSize;
-    line-height: 1;
-    margin: 0;
+    color: $black;
+    font-weight: $fontWeightBlack;
 
     &--small {
-      font-size: $headerPrimarySmallFontSize;
+      @include typeVariant(headline);
     }
 
     &--light {
-      color: $headerColorLight;
+      color: $white;
     }
-
   }
+
   .mint-header-secondary {
     @include component;
+    @include typeVariant(default, 0.667);
     display: block;
-    color: $headerColor;
-    font-family: $headerSecondaryFontFamily;
-    font-size: $headerSecondaryFontSize;
-    line-height: 1;
+    color: $black;
+    font-weight: bold;
+    letter-spacing: 0.03em;
```

Comment:
Tiny letter-spacing usually make condensed UPPERCASE letters more readable.
We have those in some mockups, but we need to revisit some components